testing/python/language/test_rand.py

```
@@ -0,0 +1,39 @@
+tilelang.disable_cache()
+
+@tilelang.jit
+def tilelang_rand_1d(M=1024, seed=42):
+
+    blk_M = 128
+    num_threads = 128
+
+    @T.prim_func
+    def rand_kernel(A: T.Tensor((M,), 'uint32')):
+        with T.Kernel(M // blk_M, threads=num_threads) as bx:
+            tx = T.get_thread_binding()
```

Ruff (0.14.8)

Local variable `tx` is assigned to but never used

Remove assignment to unused variable `tx`

(F841)